### PR TITLE
Fix for 1468

### PR DESCRIPTION
**Pull request by sbrossie**

### CODE REVIEW

File: usage/src/main/java/org/killbill/billing/usage/plugin/api/UsagePluginApi.java [hunk 1]
```diff
@@ -21,12 +21,12 @@
...
 
 import org.joda.time.LocalDate;
+import org.killbill.billing.payment.api.PluginProperty;
 import org.killbill.billing.usage.api.RawUsageRecord;
-import org.killbill.billing.util.callcontext.TenantContext;
 
 public interface UsagePluginApi {
 
-    List<RawUsageRecord> getUsageForAccount(final LocalDate startDate, final LocalDate endDate, final TenantContext context);
+    List<RawUsageRecord> getUsageForAccount(final LocalDate startDate, final LocalDate endDate, final UsageContext context, Iterable<PluginProperty> properties);
```
Comment (pierre) [inline usage/src/main/java/org/killbill/billing/usage/plugin/api/UsagePluginApi.java:29]: Nit: either `final` throughout or none at all.

File: .idea/encodings.xml [hunk 2]
```diff
@@ -4,30 +4,20 @@
...
     <file url="file://$PROJECT_DIR$/notification/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/notification/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/payment" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/payment/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/payment/src/main/resources" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/usage" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/usage/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/usage/src/main/resources" charset="UTF-8" />
```
Comment (pierre) [inline .idea/encodings.xml:30]: Are you on the latest version of IntelliJ? I had just fixed this on 5e1c8d53c7fb85a89e89c1512f4d49acce29e0cf.

**Review by pierre — APPROVED** [review]

File: usage/src/main/java/org/killbill/billing/usage/plugin/api/UsagePluginApi.java (hunk 1 above)
Comment (sbrossie) [reply usage/src/main/java/org/killbill/billing/usage/plugin/api/UsagePluginApi.java]: Fixed in https://github.com/killbill/killbill-plugin-api/pull/21/commits/588163af3a06752b46ef448f1c76519b16f9a790

File: .idea/encodings.xml (hunk 2 above)
Comment (sbrossie) [reply .idea/encodings.xml:30]: Screen Shot 2022-03-31 at 3 35 08 PM

**Pull request merged**